command/update/command.go

```diff
@@ -158,10 +157,12 @@ func (c *Command) execute() error {
 	}
 
 	// Here we lookup the VM IP we are interested in.
-	var podInfos []provider.PodInfo
+	var podInfo provider.PodInfo
 	{
 		action := func() error {
-			podInfos, err = newProvider.Lookup()
+			podInfo, err = newProvider.Lookup()
```

Comment:
I think it would make sense to only return the IP since nothing else is now of interest anymore.